core/test/unit/models/base/utils_spec.js

```diff
@@ -0,0 +1,25 @@
+var should = require('should'), // jshint ignore:line
+    sinon = require('sinon'),
+    utils = require('../../../../server/models/base/utils');
+
+describe('Models: Utils', function () {
+    describe('actorIs', function () {
+        it('returns false when user not defined', function () {
+            should(Boolean(utils.actorIs(null, 'Author'))).be.false();
+        });
+
+        it('returns correctly when passed a single role', function () {
+            var user = {roles: [{name: 'Author'}]};
```

Comment:
You can use `testUtils.permissions.author`.

Reply:
ah yeah - when I added the testUtils.permissions stuff I forgot to go back and change these tests.

Reply:
cool thanks 👍